01.fizzbuzz/fizzbuzz.rb

```diff
@@ -1,16 +1,13 @@
 #!/bin/env ruby
-x = 1
-while x <= 20
-  case
-  	when x % 3 == 0 && x % 5 != 0
-  		puts "Fizz"
-  	when x % 3 != 0 && x % 5 == 0
-  		puts "Buzz"
-  	when x % 3 ==0  && x % 5 == 0
- 			puts "FizzBuzz"
- 		else
- 			puts x
-	end
-	x += 1
+nums = 1..20
+nums.each do |n|
```

Comment:
これぐらいであれば、変数に入れない、というアプローチもあります。

```suggestion
(1..20).each do |n|
```